github/messages.go

```diff
@@ -191,5 +191,5 @@ func ParseWebHook(messageType string, payload []byte) (interface{}, error) {
 		Type:       &eventType,
```

Comment:
You also missed a mention of old function name in the docs of this func.

It should be:

> as returned by Event.ParsePayload()

Rather than the current:

> as returned by Event.Payload()